### PR TITLE
Fix Operators + Clean Up Files

### PR DESCRIPTION
**Pull request by sidpremkumar**
Remove cmake-build-debug folder and clean up operators so all of them are working

### CODE REVIEW

File: src/leodb/db.cpp [hunk 1]
```diff
@@ -12,6 +14,7 @@ bool DB<T, U>::put(Key<T> key, Value<U> value) {
     try {
         int inserted = key.hashItem();
         table[inserted] = Entry<T, U>(key, value);
+        totalKeys += 1;
```
Comment (emampons) [inline src/leodb/db.cpp:16]: duplicate entries shouldn't increment the count

File: src/leodb/db.cpp [hunk 2]
```diff
@@ -12,6 +14,7 @@ bool DB<T, U>::put(Key<T> key, Value<U> value) {
     try {
```
Comment (emampons) [inline src/leodb/db.cpp]: try catch logic doesn't make sense since it doesnt error out

**Review by emampons — CHANGES_REQUESTED** [review]
catch/try logic doesn't make sense

File: src/leodb/db.cpp (hunk 2 above)
Comment (sidpremkumar) [reply src/leodb/db.cpp]: Removed

File: src/leodb/db.cpp (hunk 1 above)
Comment (sidpremkumar) [reply src/leodb/db.cpp:16]: Yup added: 
```
if (table.count(inserted) == 0) {
        totalKeys += 1;
    }
```

**Review by emampons — APPROVED** [review]

**Pull request merged**